static/app/components/forms/selectOption.tsx

```diff
@@ -3,14 +3,20 @@ import styled from '@emotion/styled';
 
 import {IconCheckmark} from 'sentry/icons';
 import space from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
 
 type Props = React.ComponentProps<typeof selectComponents.Option>;
 
 function SelectOption(props: Props) {
+  const {data} = props;
```

Comment:
is there a reason we don't destructure more props in here like isFocus, isMulti?

Reply:
good point, just did!